tophub/cuda_v0.04.log

```diff
@@ -483,3 +483,207 @@
+{"i": ["cuda -model=tx2", "topi_nn_conv2d", [["TENSOR", [1, 128, 14, 14], "float32"], ["TENSOR", [256, 128, 1, 1], "float32"], [2, 2], [0, 0], [1, 1], "NCHW", "float32"], {}, ["conv2d", [1, 128, 14, 14, "float32"], [256, 128, 1, 1, "float32"], [2, 2], [0, 0], [1, 1], "NCHW", "float32"], {"i": 30887, "t": "direct", "c": null, "e": [["tile_f", "sp", [4, 4, 16, 1]], ["tile_y", "sp", [1, 1, 1, 7]], ["tile_x", "sp", [1, 1, 7, 1]], ["tile_rc", "sp", [16, 8]], ["tile_ry", "sp", [1, 1]], ["tile_rx", "sp", [1, 1]], ["auto_unroll_max_step", "ot", 512], ["unroll_explicit", "ot", 0]]}], "r": [[3.7923299999999997e-05], 0, 10.342200517654419, 1563866904.3233705], "v": 0.1}
+{"i": ["cuda -model=tx2", "topi_nn_conv2d", [["TENSOR", [1, 256, 7, 7], "float32"], ["TENSOR", [512, 256, 1, 1], "float32"], [2, 2], [0, 0], [1, 1], "NCHW", "float32"], {}, ["conv2d", [1, 256, 7, 7, "float32"], [512, 256, 1, 1, "float32"], [2, 2], [0, 0], [1, 1], "NCHW", "float32"], {"i": 1089916, "t": "direct", "c": null, "e": [["tile_f", "sp", [8, 4, 16, 1]], ["tile_y", "sp", [1, 2, 2, 1]], ["tile_x", "sp", [1, 1, 4, 1]], ["tile_rc", "sp", [16, 16]], ["tile_ry", "sp", [1, 1]], ["tile_rx", "sp", [1, 1]], ["auto_unroll_max_step", "ot", 1500], ["unroll_explicit", "ot", 1]]}], "r": [[3.8659300000000004e-05], 0, 12.390298128128052, 1563871214.8534417], "v": 0.1}
+{"i": ["cuda -model=tx2", "topi_nn_conv2d", [["TENSOR", [1, 512, 4, 4], "float32"], ["TENSOR", [512, 512, 3, 3], "float32"], [1, 1], [1, 1], [1, 1], "NCHW", "float32"], {}, ["conv2d", [1, 512, 4, 4, "float32"], [512, 512, 3, 3, "float32"], [1, 1], [1, 1], [1, 1], "NCHW", "float32"], {"i": 1603405, "t": "direct", "c": null, "e": [["tile_f", "sp", [8, 1, 64, 1]], ["tile_y", "sp", [1, 1, 2, 2]], ["tile_x", "sp", [1, 1, 2, 2]], ["tile_rc", "sp", [128, 4]], ["tile_ry", "sp", [1, 3]], ["tile_rx", "sp", [1, 3]], ["auto_unroll_max_step", "ot", 512], ["unroll_explicit", "ot", 0]]}], "r": [[0.0004848529], 0, 10.855116605758667, 1563873696.3061104], "v": 0.1}
+{"i": ["cuda -model=nano", "topi_nn_conv2d", [["TENSOR", [1, 3, 224, 224], "float32"], ["TENSOR", [16, 3, 3, 3], "float32"], [2, 2], [1, 1], [1, 1], "NCHW", "float32"], {}, ["conv2d", [1, 3, 224, 224, "float32"], [16, 3, 3, 3, "float32"], [2, 2], [1, 1], [1, 1], "NCHW", "float32"], {"i": 23510223, "t": "direct", "c": null, "e": [["tile_f", "sp", [1, 2, 4, 2]], ["tile_y", "sp", [112, 1, 1, 1]], ["tile_x", "sp", [1, 7, 16, 1]], ["tile_rc", "sp", [3, 1]], ["tile_ry", "sp", [1, 3]], ["tile_rx", "sp", [3, 1]], ["auto_unroll_max_step", "ot", 512], ["unroll_explicit", "ot", 1]]}], "r": [[0.00017527783194549583], 0, 2.204166889190674, 1563213049.8776293], "v": 0.1}
```

Comment:
`nano` is not as unambiguous as `tx2`. Please rename it to `jetson-nano`

Reply:
Okay made the change.